### ISSUE TITLE
Design

### ISSUE THREAD

**Issue by mvl22** [cyclestreets/android#80]
The app has never really had a 'design' as such - presumably it uses the standard Android design stuff.

I think we should aim to get a really beautiful (but functional) and innovative design, somehow.

**Comment by mvl22** [comment]
Design proposal circulated to the mobile@ list.

**Comment by jf1** [comment]
@mvl22 if you're looking for any extra input then please forward the email.

@oliverlockwood many thanks for all your recent efforts on the android app.

**Comment by mvl22** [comment]
> if you're looking for any extra input then please forward the email.

Yes, great! Just subscribed you. Will forward the e-mail when I can get to my e-mail.

**Comment by oliverlockwood** [comment]
@jf1 my pleasure 👍

**Comment by mvl22** [comment]
@smsm1 Can I add you to the list also?

**Comment by smsm1** [comment]
Could you forward me the email please. I got auto-unsubcribed from the mobile mailing list due to my mail server rejecting the spam.
